gradle/dependencies.gradle

```diff
@@ -25,26 +25,38 @@ ext {
   defaultScalaVersions = [ '2.11', '2.12' ]
   // Available if -PscalaVersion is used. This is useful when we want to support a Scala version that has
   // a higher minimum Java requirement than Kafka. This was previously the case for Scala 2.12 and Java 7.
-  availableScalaVersions = [ '2.11', '2.12' ]
+  availableScalaVersions = [ '2.11', '2.12', '2.13' ]
```

Comment:
Since 2.13 is out now, should we have this in the `defaultScalaVersions` instead?

Reply:
Questions and a note:

 **Question:** '_instead_' or '_also_' ? My assumption is that you want to include 2.13 into `defaultScalaVersions` (but `availableScalaVersions` should keep version 2.13, right ?)

**Note:** if we add 2.13 into `defaultScalaVersions` gradle tasks **_testAll_**, **_jarAll_** and **_releaseTarGzAll_** will fail until code is refactored for 2.13 (but then again: maybe this is what you want).

Reply:
I meant "also", sorry.

Reply:
Done, 2.13 is added into `defaultScalaVersions`